styles/main.css

```diff
@@ -2,3 +2,97 @@
 :root {
   --bs-font-sans-serif: 'Roboto', system-ui, -apple-system, "Segoe UI", "Helvetica Neue", Arial, "Noto Sans", "Liberation Sans", sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji";
 }
+
+
+div {
+  display: block;
+}
+
+/********************/
+/*    Customers     */
+/********************/
+
+.blue {
+  color: #00cec9;
+}
+.img-fluid {
+  max-width: 100%;
+  height: auto;
+  margin-left: auto;
+  display: block;
+}
+
+.slider-1 {
+  padding-top: 2rem;
+  padding-bottom: 3.75rem;
+  background: linear-gradient(rgb(238, 238, 238), rgb(144, 173, 198));
```

Comment:
I think this looks better without the gradient, but I don't know about everyone else.

@Future-Leaders-SI653/development what do you guys think?

```suggestion
```

Reply:
Ready!